lib/bummr/updater.rb

```diff
@@ -31,11 +36,15 @@ def update_gem(gem, index)
       end
 
       log "Commit: #{message}".color(:green)
-      system("git commit -am '#{message}'")
+      system("git add Gemfile Gemfile.lock")
+      system("git commit -m '#{message}'")
     end
 
     def updated_version_for(gem)
-      `bundle list | grep " #{gem[:name]} "`.split('(')[1].split(')')[0]
+      begin
+        `bundle list | grep " #{gem[:name]} "`.split('(')[1].split(')')[0]
+      rescue Error
+      end
```

Comment:
This throws an error when there is not an updated version (when you are updating dependencies)